src/plugins/azgenerator/TemplateAzureCliCustom.ts

```diff
@@ -65,66 +64,57 @@ function GetCommandBody(model: CodeModelAz, required: boolean, needUpdate: boole
     // method
     //
 
-    //let updatedMethodName = ((methodName != "show") ? methodName : "get").replace(/-/g, '_');
     let updatedMethodName: string = model.Command_FunctionName;
     if (needUpdate) {
-        updatedMethodName = model.Command_FunctionName.replace(/_create/g, "_update");
+        updatedMethodName = updatedMethodName.replace(/_create/g, "_update");
```

Comment:
i think that should be done in the model

Reply:
We can't split createorupdate into two operations create and update operation in codemodel. it will cause one operation has the operation definition and the other one is just has a pointer referred to the previous one.